### PR TITLE
feat(astro): add backtick string injections

### PR DESCRIPTION
**Pull request by virchau13**
This also updates a highlight to be more accurate and bumps the lockfile to fix a few bugs.

### CODE REVIEW

File: queries/astro/highlights.scm [hunk 1]
```diff
@@ -19,3 +19,9 @@
 ((erroneous_end_tag
   (erroneous_end_tag_name) @type)
   (#lua-match? @type "^[A-Z]"))
+
+(start_tag
+  (fragment_tag_name) @tag.delimiter)
```
Comment (lucario387) [inline queries/astro/highlights.scm:24]: tag.delimiter is for `<`, `>` and the like, and fragment is, I'd assume, `</>`? So naming it as tag_name and then capturing with tag.delimiter seems a bit off

**Comment by virchau13** [comment]
Yeah that probably wasn't best-practices. Fixed.

**Comment by clason** [comment]
(may want to rebase on master)

**Comment by virchau13** [comment]
Rebased.

**Review by lucario387 — APPROVED** [review]

**Pull request merged**